src/Lyrics.h

```diff
@@ -102,8 +101,8 @@ class LyricsPanel final
    void SetLyricsStyle(const LyricsStyle newLyricsStyle);
 
    void Update(double t);
-   void UpdateLyrics(wxEvent &e);
-   void UpdateLyrics();
+   void UpdateLyrics(struct UndoRedoMessage);
```

Comment:
The struct keyword is not needed.

Reply:
Incorrect, actually.  Try removing `struct`.  It won't compile.

It will compile without `struct` only if a previous line in the header has
```
struct UndoRedoMessage;
```

As I wrote it, it is an example of an "elaborated type" acting as a forward declaration introducing the type name into the compilation unit.

Reply:
I see.